### PR TITLE
Fix configuration documentation for email duration limiter

### PR DESCRIPTION
**Pull request by shimi9276**

### CODE REVIEW

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -112,8 +112,8 @@ This reference uses `.` to denote the nesting of values.
 + `email.password` `(string)` - A string representing the password of the account. It's recommended to provide this value at runtime from a secret vault of some sort.
 + `email.local_name` `(string)` - A string representing the hostname sent to the SMTP server with the HELO command. By default, "localhost" is sent.
 + `email.sender` `(string)` - A string representing the email account which is set as the sender.
-+ `email.limit_every_duration` `(duration : 1m)` - Average time to wait between sending emails. The zero value means no duration and therefore no emails can be sent.
-+ `email.burst` `(int: 10)` - Maximal burst of emails before applying `limit_every`.
++ `email.limit_every_duration` `(duration : 1m)` - Average time to wait between sending emails. The zero value means *infinite* duration and therefore *unlimited* emails can be sent.
```
Comment (nopcoder) [inline docs/reference/configuration.md:115]:
```suggestion
+ `email.limit_every_duration` `(duration : 1m)` - The average time between sending emails. If zero is entered, there is no limit to the amount of emails that can be sent.
```

File: docs/reference/configuration.md [hunk 2]
```diff
@@ -112,8 +112,8 @@ This reference uses `.` to denote the nesting of values.
...
++ `email.burst` `(int: 10)` - Maximal burst of emails before applying `limit_every`. The zero value means no burst and therefore no emails can be sent.
```
Comment (nopcoder) [inline docs/reference/configuration.md:116]: What is limit_every?

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: docs/reference/configuration.md (hunk 1 above)
Comment (shimi9276) [reply docs/reference/configuration.md]: Done

File: docs/reference/configuration.md (hunk 2 above)
Comment (shimi9276) [reply docs/reference/configuration.md]: It's referring to `limit_every_duration`  (in the previous line)

**Review by nopcoder — APPROVED** [review]

**Pull request merged**